Daniel Warrick/functions_basic_2.py

```diff
@@ -0,0 +1,42 @@
+def countdown(x):
+    z = range(x,0,-1)
+    for i in z:
+        print(i)
+
+countdown(9)
+
+def printRet(x):
+    print(x[0])
+    return x[1]
+print(printRet([1,2]))
+
+def firstLength(x):
+    print(len(x) + x[0])
+firstLength([1,2,3,4,5])
+
+def greaterThan(x):
+    y = []
+    for i in range(len(x)):
+        if len(x) < 1:
```

Comment:
Moving the if statement out of the for loop will make it computationally more efficient. Here if an array has 1000 elements, the we will be calculating the Len 1000 times and comparing it with 1 1000 times